sbin/spark-config.sh

```diff
@@ -19,21 +19,12 @@
 # should not be executable directly
 # also should not be passed any arguments, since we need original $*
 
-# resolve links - $0 may be a softlink
```

Comment:
I have trouble parsing what this script was doing before, but the desired outcome seems clear. The only change here is that `SPARK_PREFIX` is no longer set and exported, but I believe you've correctly removed usages of it in favor of `SPARK_HOME`. Good cleanup and standardization IMHO.